packages/presentation/src/components/SpaceInfo.svelte

```diff
@@ -13,18 +14,42 @@
 // limitations under the License.
 -->
 <script lang="ts">
-  import { IconFolder, IconSize, Label } from '@hcengineering/ui'
-
   import { Space } from '@hcengineering/core'
+  import { Asset } from '@hcengineering/platform'
+  import {
+    AnySvelteComponent,
+    Icon,
+    IconFolder,
+    IconSize,
+    IconWithEmoji,
+    Label,
+    getPlatformColorDef,
+    themeStore
+  } from '@hcengineering/ui'
+  import view, { IconProps } from '@hcengineering/view'
+
   import presentation from '..'
+  import { ComponentType } from 'svelte'
 
-  export let value: Space
+  export let value: Space & IconProps
   export let subtitle: string | undefined = undefined
   export let size: IconSize
+  export let iconWithEmoji: AnySvelteComponent | Asset | ComponentType | undefined = view.ids.IconWithEmoji
+  export let defaultIcon: AnySvelteComponent | Asset | ComponentType | undefined = undefined
 </script>
 
 <div class="flex-presenter">
-  <div class="icon medium-gap"><IconFolder {size} /></div>
+  <div class="icon medium-gap">
+    <Icon
+      {size}
+      icon={value.icon === iconWithEmoji && iconWithEmoji ? IconWithEmoji : value.icon ?? defaultIcon ?? IconFolder}
```

Comment:
We should do something with this hell in future)

Reply:
Yes, in the other PR I added a separate component for this. Maybe we can move it to the platform, at least.